Exercise_04_09_new.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import seaborn as sns
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[ ]:
+
+
+####1####
+
+
+# In[3]:
+
+
+data=pd.read_csv("C:/Users/kphom4872/Desktop/Training/satday-4-sept-quiz/bike_sharing_hours.csv")
+
+
+# In[4]:
+
+
+data.shape
+
+
+# In[5]:
+
+
+data.head(2)
+
+
+# In[75]:
+
+
+df1=data.loc[data["mnth"] <= 3]
+
+
+# In[76]:
+
+
+df1.head(2)
+
+
+# In[77]:
+
+
+df2=pd.DataFrame(data.groupby(['mnth'])["temp"].mean())
+
+
+# In[78]:
+
+
+df2
+
+
+# In[79]:
+
+
+quat_1=df2[0:3].mean()
+quat_1
+
+
+# In[80]:
+
+
+quat_3=df2[6:9].mean()
+quat_3
+
+
+# In[69]:
+
+
+from scipy.stats import ttest_1samp
+
+
+# In[81]:
+
+
+ttest_1samp(df1['temp'],popmean=quat_1)
+
```

Comment:
should you not be comparing this with 3rd quarter data ?

Reply:
I have fixed the 1st quarter average and then compared it to 3rd data.